flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -203,27 +202,27 @@ def __init__(
                         "modifiers"
                     ].get()
                     raise ValueError(
-                        "Not implemented yet"
+                        "This feature has not been implemented yet."
                     )  # TODO create a Stacked from all
```

Comment:
I think a `NotImplementedError` exception would be a bit more appropriate than a `ValueError`?